### ISSUE TITLE
Model training

### ISSUE THREAD

**Issue by Juanfra21** [Juanfra21/text2sql#6]
1. Train model using the training set and validate using the validation set to tune hyperparameters.
2. High training time with the amount of data we have. Might be a good idea to randomly sample (Incremental steps).
